### DISCUSSION TITLE
Message received with Double quotes data=b'"I can help!"'

### DISCUSSION THREAD

**Discussion by khanakia** [nats-io/nats.py#516] [Q&A]
I am using Golang as a Subscriber and Python as a Client. Response adds `"I can help!"` the double quotes how do I get the response without Double Quotes?

Golang Code
```go
c.Subscribe("help", func(subj, reply string, msg string) {
  c.Publish(reply, "I can help!")
})
```

Python Client
```py
import asyncio
import nats

async def main():
    nc = await nats.connect()
    resp = await nc.request('help', b'help', 5) 
    print(resp.data.decode())
    await nc.close()

if __name__ == '__main__':
    asyncio.run(main())
```

## Response
```
Msg(_client=<nats client v2.6.0>, subject='_INBOX.YhKpg18ihR7EqGXm6QE5Gr.YhKpg18ihR7EqGXm6QE5Lh91f1', 
reply='', data=b'"I can help!"', headers=None, _metadata=None, _ackd=False, _sid=1)
```

**Comment by wallyqs** [comment]
Hi, which Go client are you using? the callback looks a bit different, maybe try []byte(“I can help”)

**Reply by khanakia** [reply]
I am using https://github.com/nats-io/nats.go

I tried using bytes then it returned the below response

```py
nc = await nats.connect()
  resp = await nc.request('help', b'help', 5) 
  msg = resp.data.decode("utf-8")    
  print(str(msg))

```
outputs: "SSBjYW4gaGVscCE="

**Reply by khanakia** [reply]
Another note i am using`*nats.EncodedConn` not `*nats.Conn` may be that's the reason.

**Reply by khanakia** [reply]
I was able to remove the quotes using below code but I am not sure if that's the right way

```
import ast
print(ast.literal_eval(msg))
```

output: `I can help!`
